src/pages/gd/Bridge/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { memo } from 'react'
+import useActiveWeb3React from 'hooks/useActiveWeb3React'
+import { useApplicationTheme } from 'state/application/hooks'
+import {
+    KimaTransactionWidget,
+    KimaProvider,
+    FontSizeOptions,
+    ColorModeOptions,
+    ModeOptions,
+    DAppOptions,
+} from '@kimafinance/kima-transaction-widget'
+import '@kimafinance/kima-transaction-widget/dist/index.css'
+
+const Bridge = memo(() => {
```

Comment:
@L03TJ3 we should probably actually move this widget to good-design

Reply:
The reasoning we had for the ube-swap component is that since there is close to no customization styling it does not make that much sense tbh
It does not give any benefit, except complications on how to handle the events / provider etc

So I don't see the need for it
@johnsmith-gooddollar wdyt?

Reply:
The reasoning we had for the ube-swap component is that since there is close to no customization styling it does not make that much sense tbh
It does not give any benefit, except complications on how to handle the events / provider etc

So I don't see the need for it
@johnsmith-gooddollar wdyt?

Reply:
ok